lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Network.hs

```diff
@@ -46,4 +48,5 @@ spec = do
             [ expectField (#decentralizationLevel) (`shouldBe` d)
             , expectField (#desiredPoolNumber) (`shouldBe` nOpt)
             , expectField (#minimumUtxoValue) (`shouldBe` minUtxoValue)
+            , expectField (#hardforkEpochInfo) (`shouldNotBe` Nothing)
```

Comment:
Will this one be ever true? Since we only run the test scenarios AFTER the hard-fork has happened, I'd assume that `Nothing` will be returned forever. No ?

Reply:
it was Nothing...if you run this test once! ie., one has to wait for tip client waiting to make local query
```
stack test --ta '-m "NETWORK_PARAMS - Able to fetch network parameters"' cardano-wallet-shelley:integration 
```
But when running several tests then I have Just ... so I used eventually